### PR TITLE
Laird WiFi

### PR DESCRIPTION
**Pull request by matinlotfali**
This pull request adds a layer for Laird Sterling LWB5+ WiFi module

### CODE REVIEW

**Review by mistyron — CHANGES_REQUESTED** [review]
Please add a description about the changes in this PR on the Conversation tab

File: .gitmodules [hunk 1]
```diff
@@ -1,3 +1,6 @@
 [submodule "Build/meta-mistysom"]
 	path = Build/meta-mistysom
 	url = https://github.com/MistySOM/meta-mistysom.git
+[submodule "Build/meta-mistylwb5p"]
```
Comment (mistyron) [inline .gitmodules:4]: mind thart the fork is out of date, probably should/could be updated.

**Review by mistyron — COMMENTED** [review]
Does forked Laird repo need to be updated?

File: .gitmodules (hunk 1 above)
Comment (matinlotfali) [reply .gitmodules:4]: When reading the difference, I can see that it only affects the bluetooth on lwb5p and not WiFi.

**Review by mistyron — APPROVED** [review]
LGTM

**Pull request merged**